content/documentation/runtimes-monitoring/index.adoc

```diff
@@ -103,6 +103,17 @@ As an example and for self-monitoring purpose Kiali itself link:https://github.c
 
 The pod annotation for Kiali is: `kiali.io/runtimes: go`
 
+
+=== Envoy
+
+The Envoy sidecars that come with Istio are also supplying link:https://www.envoyproxy.io/docs/envoy/latest/configuration/upstream/cluster_manager/cluster_stats[some internal metrics], that you can see in Kiali. They are not the same than the metrics reported by Istio Telemetry, and of which Kiali makes an intensive usage. Some of them can be redundant.
+
+Unlike the other Runtimes dashboards, this one has to be enabled explicitely with the pod annotation `kiali.io/runtimes: envoy` (there's no automatic discovery configured).
+
+Note that the enabled Envoy metrics can be tuned, as explained in link:https://istio.io/docs/ops/telemetry/envoy-stats/[Istio documentation]: it's possible to get more metrics using the `statsInclusionPrefixes` annotation.
+
+E.g. `sidecar.istio.io/statsInclusionPrefixes: cluster_manager,listener_manager,listener` will add `listener` metrics to the usual ones, for more inbound traffic information. You can then customize the Envoy dashboard of Kiali according to the collected metrics.
```

Comment:
Replace "E.g." with "For example"

Also this example needs a bit more information, do you need all three (cluster_manager,listener_manager,listener) to add listener metrics?  The Istio docs say "To gather data on inbound traffic, add the listener prefix.", but without clicking over there, I would not have known that the other two settings were defaults.